tasks/eth/005-2-ecotone-set-gas-config/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+FOUNDATION_SAFE="0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A"
```

Comment:
We could remove this with the suggested fix above, but I say we keep it for clarity.